docs/source/test_network.md

```diff
@@ -0,0 +1,578 @@
+```
+
+If the command is successful, you should see the following response:
+```
+2019-12-04 17:38:21.048 EST [chaincodeCmd] chaincodeInvokeOrQuery -> INFO 001 Chaincode invoke successful. result: status:200
+```
+
+Because the endorsement policy for the fabcar chaincode requires you to get
+signatures from both org1 and org2, you need to provide the peer endpoints and
+TLS certificates for `peer0.org1.example.com` and `peer0.org1.example.com` to
+the `--peerAddresses` and `--tlsRootCertFiles` flags:
+```
+ --peerAddresses localhost:7051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt --peerAddresses localhost:9051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
+```
+
+You also set the following environment variables to operate as `org2.example.com`:
+```
+# Environment variables for org1.example.com
+
+export CORE_PEER_TLS_ENABLED=true
+export CORE_PEER_LOCALMSPID="Org2MSP"
+export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
+export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp
+export CORE_PEER_ADDRESS=localhost:9051
+```
+
+Operating as Org2, you can now query the fabcar chaincode running on
+`peer0.org2.example.com`:
+```
+peer chaincode query -C mychannel -n fabcar -c '{"Args":["queryAllCars"]}'
+```
+The result will show the changed car owner from the invoke command:
+```
+[{"Key":"CAR0", "Record":{"make":"Toyota","model":"Prius","colour":"blue","owner":"Dave"}},{"Key":"CAR1", "Record":{"make":"Ford","model":"Mustang","colour":"red","owner":"Brad"}},{"Key":"CAR2", "Record":{"make":"Hyundai","model":"Tucson","colour":"green","owner":"Jin Soo"}},{"Key":"CAR3", "Record":{"make":"Volkswagen","model":"Passat","colour":"yellow","owner":"Max"}},{"Key":"CAR4", "Record":{"make":"Tesla","model":"S","colour":"black","owner":"Adriana"}},{"Key":"CAR5", "Record":{"make":"Peugeot","model":"205","colour":"purple","owner":"Michel"}},{"Key":"CAR6", "Record":{"make":"Chery","model":"S22L","colour":"white","owner":"Aarav"}},{"Key":"CAR7", "Record":{"make":"Fiat","model":"Punto","colour":"violet","owner":"Pari"}},{"Key":"CAR8", "Record":{"make":"Tata","model":"Nano","colour":"indigo","owner":"Valeria"}},{"Key":"CAR9", "Record":{"make":"Holden","model":"Barina","colour":"brown","owner":"Shotaro"}}]
+```
+
+## Bring up the network with Certificate Authorities
+
+Hyperledger Fabric uses public key infrastructure (PKI) to verify the actions of
```

Comment:
These "bring up the network" sections are organized somewhat strangely to me, especially this one after the CLI section above. I might put all of these sections together, either before or after the CLI section, and have links at the top of the doc that can take people down to these "optional deployment" sections.

Reply:
I debated this in terms of putting it above the channel creation, but I wanted to create a friendly experience where users were not hit by too much information to quickly. I will put a link to this section from the top however.